Sdl.Community.Toolkit.LanguagePlatform/Models/PathInfo.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+
+namespace Sdl.Community.Toolkit.LanguagePlatform.Models
+{
+	/// <summary>
+	/// Environment path info
+	/// </summary>
+	public class PathInfo
+	{
+		private string _productPath;
+		private string _tmPath;
+		private readonly string _productName;
+
+		/// <summary>
+		/// Path Info
+		/// </summary>
+		/// <param name="productName"></param>
+		public PathInfo(string productName)
+		{
+			_productName = productName;
+		}
+		//
+
+		// Summary:
+		//     The default product path, created in the my documents folder
+		public virtual string ProductPath
+		{
+			get
+			{
+				if (!string.IsNullOrEmpty(_productPath))
+					return _productPath;
+
+				_productPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), _productName);
```

Comment:
Shouldn't we use the Studio folder path? The toolkit is not a product. The toolkit core has a bunch of libraries that provide the information required for identifying current version of Studio.

Reply:
The path information where the temporary resources are created and managed to provide this functionality is the responsibility of the client that is integrating with the functionality offered by the toolkit e.g. thier product. The class called PathInfo includes default paths (and names), however this is the developers choice.

Wouldn’t be more correct to allow the developer to decide where these temporary resources are created & managed, with the expected defaults?

Reply:
Just thinking this through again and might make more sense to manage the temporary resources used in providing the functionality in the Studio folder path as you suggested.  I'll update the PR later on this evening

Reply:
Thanks Patrick. I agree that the developer should be able to decide about the path so having the PathInfo virtual is great. It's just around the default value.